setup.py

```diff
@@ -26,7 +26,8 @@
             'deepcell @ git+https://github.com/AllenInstitute/DeepCell.git'
         ],
         'deepinterpolation': [
-            'deepinterpolation @ git+https://github.com/AllenInstitute/deepinterpolation'   # noqa E401
+            'tensorflow==2.11.0',
+            'deepinterpolation @ git+https://github.com/AllenInstitute/deepinterpolation_ai_internal@refactor'   # noqa E401
```

Comment:
note I will update the branch once the deepinterpolation pr is merged to the main branch